composer.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "bcgov-theme/design-system-wordpress-child-theme-ti-corp",
+    "description": "Design System Wordpress Child Theme TI-Corp",
+    "version": "1.0.0",
+    "type": "wordpress-theme",
+    "license": "Apache-2.0",
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://apps.itsm.gov.bc.ca/satis"
+        }
+    ],
+    "require-dev": {
+        "bcgov/wordpress-utils": "@dev"
```

Comment:
this really should be a version, we got caught doing this before, it just helps if we need to make breaking changes.